lib/vrl/compiler/src/enrichment_tables.rs

```diff
@@ -0,0 +1,36 @@
+use dyn_clone::DynClone;
```

Comment:
nit: should this be moved to the follow-up PR at https://github.com/timberio/vector/pull/8660 (might make it easier for me and others to review the VRL changes in its entirety)?

Reply:
Ah, never mind, I see now that this is actually a no-op implementation to get things to compile. That seems fine to me 👍